tree/dataframe/src/RCutFlowReport.cxx

```diff
@@ -49,6 +49,28 @@ const TCutInfo &RCutFlowReport::operator[](std::string_view cutName)
    return *it;
 }
 
+void RCutFlowReport::Merge(RCutFlowReport &other)
+{
```

Comment:
as a first thing I would check that `other` is compatible with `this`: they must have cut infos for cuts with the same names

Reply:
In the new `TCutInfo::Merge` method there is a check that the two cuts must have the same name. But anyways I believe that it shouldn't be a problem since in `RCutFlowReport::Merge` the `operator[]` is called with the name of the cut as an argument. If the other report doesn't have a cut with the same name an error will be raised even before entering `TCutInfo::Merge`